doc/nrf/security/ap_protect.rst

```diff
@@ -45,16 +45,16 @@ For detailed information, refer to the hardware documentation.
 
        To keep the AP-Protect disabled, ``UICR.APPROTECT`` must be programmed to ``HwDisabled`` and the firmware must write ``SwDisable`` to ``APPROTECT.DISABLE``.
 
-The following table lists related SoCs with information about the AP-Protect mechanism they support.
-For some SoCs, the AP-Protect implementation is different depending on the build code of the device.
+The following table lists related devices with information about the AP-Protect mechanism they support.
+For some devices, the AP-Protect implementation is different depending on the build code of the device.
 See the related hardware documentation for more information about which implementation is supported for which build code and about the differences between the supported implementations.
 
-.. list-table:: SoC AP-Protect matrix
+.. list-table:: Device AP-Protect matrix
    :header-rows: 1
    :align: center
    :widths: auto
 
-   * - SoC
+   * - Device
```

Comment:
We're specifically referring to SoCs here, because of the link to hardware documentation and because otherwise we would need to list all devices, such as Thingy:91 and so on.
```suggestion
SoC or SiP